tests/e2e/framework/kubernetes.go

```diff
@@ -83,6 +86,7 @@ var (
 	rbacEnable   = flag.Bool("rbac_enable", true, "Enable rbac")
 	localCluster = flag.Bool("use_local_cluster", false,
 		"Whether the cluster is local or not (i.e. the test is running within the cluster). If running on minikube, this should be set to true.")
+	localRegistry             = flag.Bool("use_local_registry", true, "Use in-cluster docker registry for testing")
```

Comment:
Please don't enable new features by default - first they need to go trough some stability and testing, and when we confirm it works better than alternative we can switch.

Reply:
Using a local registry just suggests that a local registry should be deployed by default.  It either works or it doesn't ... and if it doesn't, the PR won't be merged. TBH, I'm not sure we even need a flag for this ... we should just always deploy an in-cluster registry.